internal/code-generator/generator/generator.go

```diff
@@ -72,3 +74,45 @@ func executeTemplate(w io.Writer, name, path, commit string, fmap template.FuncM
 	_, err = w.Write(src)
 	return err
 }
+
+// replaceConstTable defines the chars replaced on the constant generation
+// the `^` means that should be at the begining of the string.
+var replaceConstTable = map[string]string{
+	".":  "",
+	" ":  "",
+	"-":  "",
+	"'":  "",
+	"+":  "Plus",
+	"#":  "Sharp",
+	"*":  "Star",
+	"^1": "One",
+	"^2": "Two",
+	"^3": "Three",
+	"^4": "Four",
+	"^5": "Five",
+	"^6": "Six",
+	"^7": "Seven",
+	"^8": "Eight",
+	"^9": "Nine",
+	"^0": "Zero",
+}
+
+// languageConst translate an language as string on in a constant.
+func languageConst(lang string) string {
+	for i, o := range replaceConstTable {
+		if i[0] != '^' {
+			lang = strings.ReplaceAll(lang, i, o)
+			continue
+		}
+
+		if len(lang) <= 1 || i[1] != lang[0] {
+			continue
+		}
+
+		fmt.Println(o, lang[1:], lang)
```

Comment:
Probably a nit pick, but was not sure if this a debug leftover or is printing intentional here?